pkg/npm/npm_test.go

```diff
@@ -33,6 +33,10 @@ func TestYarnInstall(t *testing.T) {
 }
 
 func testInstall(t *testing.T, expectedBin string) {
+	// Skip for community PRs
+	if os.Getenv("COMMUNITY_PR") == "true" {
+		t.Skip("Skipped for community PRs")
+	}
```

Comment:
Related to the other feedback - it would be great if we could keep the codebase itself independt of the notion of "community PR" (which is really a test/CI construct).  Ideally the tests refer to semantically meaningful environment variables or parameterization - and the CI environment maps things that make sense at that layer into these in a way that makes sense.

What exactly is it that this test depends on?  Can we make the skip behaviour here more directly tied to that?

Reply:
It's tied to the `PULUMI_ACCESS_TOKEN`, as per the other feedback, which I think makes more sense. Does that make more sense?

Reply:
Actually, I'm not even sure why I added this, I've removed it.